Automattic/wp-calypso
# Reader: Transform Videopress Block embed into iframe

Fixes https://github.com/Automattic/loop/issues/163

The issue here is that the post content contains the VideoPress block. On the creators site, this block gets transformed into an iframed video but it doesn't work on Reader. This PR will add a rule to look at the dom for a VideoPress block and convert it into an iframe so that the video will be rendered on full post page on Reader.

There may be other blocks we can do this for, but best to keep this for just VideoPress for now I think.

## Proposed Changes

* This PR adds a rule to transform a Videopress Block embed markup into an iframe so that it can displayed in the reader full post content

## Before

<img width="1164" alt="Screenshot 2024-06-12 at 11 24 16" src="https://github.com/Automattic/wp-calypso/assets/5560595/59c95ee0-9e3b-4fed-8a24-04d08812523a">

## After

<img width="1199" alt="Screenshot 2024-06-12 at 11 25 04" src="https://github.com/Automattic/wp-calypso/assets/5560595/714c2f60-1163-45af-911c-f30dce1d8919">

## Why are these changes being made?


* To improve the reading experience

## Testing Instructions



* Confirm the following loads and works as expected;
* Go to '/read/blogs/9440213/posts/922361'
* Go to other posts by that user and confirm posts load with VideoPress videos.

## Pre-merge Checklist
